src/client/debugger/jupyter/helper.ts

```diff
@@ -7,7 +7,7 @@ import { IKernelDebugAdapterConfig, KernelDebugMode } from '../types';
 
 export async function isUsingIpykernel6OrLater(kernel: IKernel): Promise<boolean> {
     const code = 'import ipykernel\nprint(ipykernel.__version__)';
-    const output = await kernel.executeHidden(code);
+    const output = await kernel.queueAndExeceuteHidden(code);
```

Comment:
This should not be queued, only the debugger start method should be queued, all other (older) methods that used hidden Execution b should not have to be queued

Reply:
Not queuing that is the root of the issue